doc/plugins.rst

```diff
@@ -322,6 +322,20 @@ e.g.:
      - The Tidal "audioquality" parameter. Possible values: HI_RES, LOSSLESS, HIGH, LOW. Default is HIGH.
 
 .. _decoder_plugins:
+
+youtube-dl
+----------
+
+Play URLs supported by `youtube-dl <http://ytdl-org.github.io/youtube-dl/>`_.
+By default, it plays http/https URLs from ``youtu.be``, ``www.youtube.com``,
+``soundcloud.com``, and ``www.twitch.tv``. To play other URLs, prepend ``ytdl://``
+in front of it.
```

Comment:
So I think it'd be good to include examples for both modes of use, something more like...

```suggestion
This plugin can play links from YouTube directly. To play songs from other supported sites, prepend ``ytdl://`` in front of the URL.

.. code-block:: none

   mpc add https://www.youtube.com/watch?v=saDmN2f3HI0
   mpc add ytdl://https://www.twitch.tv/monstercat

```

If you want to be fancy, make `supported sites` an inline link there?

Also let's try to not be explicit about the whitelist sites, since they may change. I think the only one right now that we want to promise is youtube.com